github/orgs_teams.go

```diff
@@ -237,7 +242,8 @@ func (s *OrganizationsService) ListTeamRepos(ctx context.Context, team int, opt
 	}
 
 	// TODO: remove custom Accept header when topics API fully launches.
-	req.Header.Set("Accept", mediaTypeTopicsPreview)
+	headers := strings.Join([]string{mediaTypeTopicsPreview, mediaTypeNestedTeamsPreview}, ", ")
```

Comment:
nit: I prefer the way you did this at lines 269-270 below where you first make the string slice then join them on the next line, and if you wish to make the two consistent with each other, I would prefer that you change this instance...
(looking at the code, I see that the two-step method is already being done in `repos.go` and `repos_test.go`)...
but I'm fine either way and will wait to merge until I hear your opinion.

Thanks!

Reply:
@gmlewis Thanks - I fixed it.